src/wp-includes/class-wp-script-modules.php

```diff
@@ -405,15 +522,21 @@ public function print_import_map() {
 	 */
 	private function get_import_map(): array {
 		$imports = array();
-		foreach ( $this->get_dependencies( array_unique( $this->queue ) ) as $id => $script_module ) {
-			$imports[ $id ] = $this->get_src( $id );
+		foreach ( $this->get_dependencies( array_unique( $this->queue ) ) as $id ) {
```

Comment:
method returns array_keys which are unique by defintion in PHP.

```suggestion
		foreach ( $this->get_dependencies( $this->queue ) as $id ) {
```